### PR TITLE
Update v7.01 to v7.02

### PR DESCRIPTION
**Pull request by CleoMenezesJr**
Related: #16

### CODE REVIEW

**Comment by CleoMenezesJr** [comment]
bot, build

File: fm.reaper.Reaper.metainfo.xml [hunk 1]
```diff
@@ -24,6 +24,44 @@
...
+          <lib>MIDI hardware</lib>
+          <lib>Mixer</lib>
+          <lib>ReaComp</lib>
+          <lib>ReaLimit</lib>
+          <lib>Theme</lib>
+          <lib>Toolbars</lib>
+          <lib>Track panels</lib>
+          <lib>Tracks</lib>
+          <lib>Virtual MIDI Keyboard</lib>
+          <lib>Windows</lib>
```
Comment (eshimoniak-ucr) [inline fm.reaper.Reaper.metainfo.xml:61]: Should be `li` instead of `lib`

Comment (rootmeanclaire) [inline fm.reaper.Reaper.metainfo.xml:61]: Should be `li` instead of `lib`

Comment (CleoMenezesJr) [reply fm.reaper.Reaper.metainfo.xml:61]: Can you believe I spent the day wondering what was wrong? I looked a thousand times and didn't find anything strange. Hahaha thank you.

**Pull request merged**